README-ko.md

```diff
@@ -49,21 +56,26 @@ make container-serve
 
 ## Hugo를 사용하여 로컬에서 웹사이트 실행하기
 
-[`netlify.toml`](netlify.toml#L10) 파일의 `HUGO_VERSION` 환경 변수에 지정된 Hugo 확장 버전을 설치해야 합니다.
+[`netlify.toml`](netlify.toml#L11) 파일의 `HUGO_VERSION` 환경 변수에 지정된 Hugo 확장 버전을 설치해야 합니다.
 
 사이트를 로컬에서 빌드하고 테스트하려면, 다음을 실행합니다.
 
-```bash
-# 의존성 있는 소프트웨어 설치
-npm ci
-make serve
-```
+- 리눅스와 맥OS (macOS and Linux)
```

Comment:
https://github.com/kubernetes/website/blob/7b4c3a52db5117c6689624efb2cbfc3a84301356/content/ko/docs/tasks/tools/install-kubectl-macos.md?plain=1#L26

I think it would be best to denote 'macOS' exactly as in the example above.

```suggestion
- macOS와 리눅스
```